src/compile.test.ts

```diff
@@ -190,7 +222,19 @@ describe('when allowUnknownExternals is enabled', () => {
         allowUnknownExternals: true,
       }).then(getVinylContents),
     ).resolves.toMatchSnapshot();
-    expect(mockDiagnosticHandler.mock.calls).toMatchSnapshot();
+    expect(mockDiagnosticHandler.mock.calls).toEqual([
+      [
+        {
+          category: 0,
```

Comment:
nit: use the actual category enum?

Reply:
Changed to use `DiagnosticCategory.Warning` from *./diagnostics.ts*